src/main/java/gift/config/KakaoProperties.java

```diff
@@ -0,0 +1,17 @@
+package gift.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+@Component
```

Comment:
```suggestion
@EnableConfigurationProperties(KakaoProperties.class)
```
동작에 큰 차이는 없지만, 이렇게 바꾸면 설정값 주입이라는 의도가 좀 더 드러날것 같습니다~

Reply:
넵 알겠습니다! 수정하겠습니다! 감사합니다ㅎㅎ